resources/js/tasks/components/QuickFillPreview.vue

```diff
@@ -277,16 +274,21 @@ export default {
      * remove it to maintain compatibility.
      */
     reCalculateZIndex(sw) {
-      let searchBar = document.querySelector('.global-search-bar');
-      if(!searchBar){
-        return;
-      }
-      let obj = searchBar.classList;
-      if (sw === true) {
-        obj.add("global-search-bar-z-index");
-      } else {
-        obj.remove("global-search-bar-z-index");
+      let names = [".global-search-bar", ".navbar-nav"];
+      for (let i = 0; i < names.length; i++) {
```

Comment:
SonarQube is making the observation to use for (const name of names) instead of simple for